pkg/cmd/admin/policy/reconcile_clusterrolebindings.go

```diff
@@ -143,15 +147,18 @@ func (o *ReconcileClusterRoleBindingsOptions) Validate() error {
 	return nil
 }
 
-// ReconcileClusterRoleBindingsOptions contains all the necessary functionality for the OpenShift cli reconcile-cluster-role-bindings command
 func (o *ReconcileClusterRoleBindingsOptions) RunReconcileClusterRoleBindings(cmd *cobra.Command, f *clientcmd.Factory) error {
-	changedClusterRoleBindings, err := o.ChangedClusterRoleBindings()
-	if err != nil {
-		return err
+	changedClusterRoleBindings, fetchErr := o.ChangedClusterRoleBindings()
+	if IsClusterRoleBindingLookupError(fetchErr) {
+		// we got a partial match, so we want to warn the user but we can continue since there's no harm in updating the
+		// bindings that we do know need to be updated for reconciliation to be complete
+		fmt.Fprintf(o.Err, fmt.Errorf("Full reconciliation not possible: %v", fetchErr))
```

Comment:
Since we return an error, we don't need to output this here, it will show up twice, right?